public/data/viroopaksh-utrayak.json

```
@@ -0,0 +1,33 @@
+{
+  "name": "viroopaksh-utrayak",
+  "type": "personal",
+  "bio": "well I'm a programmer(java) but first of all I'm a learner",
+  "avatar": "https://github.com/viroopaksh-utrayak.png",
+  "links": [
+    {
+      "name": "Here is my Github Accnt",
```

If you want to improve this pr then:
Spelling correction
```suggestion
      "name": "Here is my Github Account",
```

---

I thought I made a comment asking to do this in my review, its gone now

---

@KendallDoesCoding Maybe the owner would prefer it says Accnt.

---

> @KendallDoesCoding Maybe the owner would prefer it says Accnt.

Yeah, True..